### ISSUE TITLE
implement inboundAddGroupOrProjectCIJobTokenScope

### ISSUE THREAD

**Issue by stephen-dahl** [jdalrymple/gitbeaker#3614]
**Description**

Gitlab recently added a new api `inboundAddGroupOrProjectCIJobTokenScope`
```
mutation inboundAddGroupOrProjectCIJobTokenScope($projectPath: ID!, $targetPath: ID!) {
  ciJobTokenScopeAddGroupOrProject(
    input: {projectPath: $projectPath, targetPath: $targetPath}
  ) {
    errors
    __typename
  }
}
```

**Proposal**

implement this api. i believe it will eventually replace gl.JobTokenScopes.addToInboundAllowList but I may be wronge.

**Checklist**

- [ x] I have checked that this is not a duplicate issue.
- [ x] I have read the documentation.

**Comment by profblackjack** [comment]
Since the above addition is the graphql api, these appear to be the REST api equivalent endpoints: https://docs.gitlab.com/ee/api/project_job_token_scopes.html

It looks like gitbeaker already has the "project" versions of the endpoints, but not the more recently introduced "groups" endpoints

**Comment by jdalrymple** [comment]
:rocket: Issue was released in [`41.0.0`](https://github.com/jdalrymple/gitbeaker/releases/tag/41.0.0) :rocket:
